xml/deployment_cifs.xml

```diff
@@ -121,6 +121,7 @@
 [<replaceable>SHARE_NAME</replaceable>]
   path = /
   vfs objects = ceph
+<!-- 2020-09-7 tbazant (ceph.conf): is the following still valid and usable? -->
```

Comment:
Ok, cifs might be different. @ddiss , do you know, if we can rely on the MON's config store here?

Reply:
@sebastian-philipp @tbazant Samba (with vfs_ceph) behaves like any other Ceph client, so a MON config store shouldn't be a problem.